angelsbioprocessing/locale/en/bio-processing.cfg

```diff
@@ -556,3 +578,50 @@ desert-garden-b=Advanced desert seed extraction
 
 swamp-garden-a=Swamp seed extraction
 swamp-garden-b=Advanced swamp seed extraction
+
+garden-cultivating=A low chance to grow gardens that you do not have yet. Not suitable for garden duplication.
+temperate-garden-cultivating-b=Duplicate a temperate garden. Suitable for perpetual generation of alien plant-life samples.
+desert-garden-cultivating-b=Duplicate a desert garden. Suitable for perpetual generation of alien plant-life samples.
+swamp-garden-cultivating-b=Duplicate a swamp garden. Suitable for perpetual generation of alien plant-life samples.
+
+[technology-description]
+
+bio-processing-brown=Brown algae are rich in sticky fibres and inorganic salts.
+bio-processing-green=Green algae are rich in robust fibres.
+bio-processing-red=Red algae are rich in calcareous compounds.
+bio-processing-blue=Blue algae are rich in nitric compounds.
+bio-processing-alien=Inoculation and processing of microbial cell cultures.
+
+bio-paper=Solid material made of organic fibres.
+
+gardens=Allows you to extract seeds and plant-life samples from gardens growing on this alien planet.
+bio-farm=Basic technology for growing and processing crops.
+bio-temperate-farming=Allows you to grow crops specialized for temperate climate conditions.
+bio-desert-farming=Allows you to grow crops specialized for dry climate conditions.
+bio-swamp-farming=Allows you to grow crops specialized for wet climate conditions.
```

Comment:
Crop specialization for X climate conditions?

Reply:
This is the technology description. Vanilla uses complete sentences for technology descriptions. I'd like to do the same. Also: with current configuration, these texts will be displayed for both levels of each tech.

Reply:
> Also: with current configuration, these texts will be displayed for both levels of each tech

you could define `localised_description` on both techs for a dedicated description?

Reply:
I don't think `localised_description` is necessary for these techs.